src/main/java/core/basesyntax/filewriter/FileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.filewriter;
+
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+
+    @Override
+    public void write(String content, String filePath) {
+        try (java.io.FileWriter writer = new java.io.FileWriter(filePath)) {
+            writer.write(content); // Записуємо текст у файл
```

Comment:
Let’s  remove uninformative comments (the same for other places, but if you want to leave it use English)